modules/quantized-mesh/docs/README.md

```diff
@@ -0,0 +1,22 @@
+# Overview
+
+The `@loaders.gl/quantized-mesh` module reconstructs mesh surfaces from the [quantized mesh](https://github.com/CesiumGS/quantized-mesh) format.
+
+## Installation
+
+```bash
+npm install @loaders.gl/quantized-mesh
+npm install @loaders.gl/core
+```
+
+## Attribution
+
+`@loaders.gl/terrain` uses [`quantized-mesh-decoder`](https://github.com/heremaps/quantized-mesh-decoder) to decode quantized mesh.
+
+Copyright (C) 2018-2019 HERE Europe B.V.
```

Comment:
I think it is enough to say that it uses the mesh loader from HERE under MIT license and have the license in the src files.